presto-main/src/main/java/com/facebook/presto/sql/planner/PlanFragment.java

```diff
@@ -199,17 +199,17 @@ private static void findRemoteSourceNodes(PlanNode node, Builder<RemoteSourceNod
 
     public PlanFragment withBucketToPartition(Optional<int[]> bucketToPartition)
     {
-        return new PlanFragment(id, root, symbols, partitioning, tableScanSchedulingOrder, partitioningScheme.withBucketToPartition(bucketToPartition), stageExecutionDescriptor, materializedExchangeSource, statsAndCosts, jsonRepresentation);
+        return new PlanFragment(id, root, symbols, partitioning, tableScanSchedulingOrder, partitioningScheme.withBucketToPartition(bucketToPartition), stageExecutionDescriptor, outputTableWriterFragment, statsAndCosts, jsonRepresentation);
```

Comment:
Every-time I see this,  I want to make it one parameter per line :P

Reply:
- What do we say to the God of the refactoring?
- Not today

😄

Reply:
Let me open a separate PR with the reformatting

Reply:
Actually I did this in https://github.com/prestodb/presto/pull/12869 :)